docs/streams/stream-basics/shared-example.md

```diff
@@ -0,0 +1,358 @@
+---
+sidebar_position: 220
+title: Shared Subscription Example
```

Comment:
@jamesm-macrometa, we might want to put the subscriptions in a subfolder of stream-basics.